GNUmakefile

```diff
@@ -29,7 +29,7 @@ check: setup
 	run-clang-tidy -extra-arg=-Wno-unknown-warning-option -p $(BUILD_DIR)/all source */source
 
 setup:
-	cd all && cmake --preset default --log-level=TRACE --trace-expand --trace-source=CPM.cmake
+	cd all && cmake --preset default #XXX --log-level=TRACE --trace-expand --trace-source=CPM_0.38.2.cmake
```

Comment:
IMHO, version number(s) shouldn't be buried within the code but prominently be placed near the top of the file.

Reply:
It is only a hint how to trace problems with `CMP_version.cmake` module. The version must be match to the `version` used at **cmake/CMP.cmake**